crates/cxx-qt-gen/src/parser/qenum.rs

```diff
@@ -96,18 +118,33 @@ mod tests {
                 Variant2,
             }
         };
+        let qenum: ItemEnum = syn::parse2(original_item.clone()).unwrap();
 
         let parsed = ParsedQEnum::parse(qenum).unwrap();
         assert_eq!(parsed.ident, "MyEnum");
+        assert_eq!(parsed.namespace, "");
```

Comment:
if there is no namespace or QObject we should error ?

Reply:
That's not done here.
The QEnum parser doesn't care whether the QEnum is assigned to anything, it's still a valid `ParsedQEnum`.

CxxQtData is responsible for matching the QEnum to the corresponding namespace or QObject.
Which will then error out if there's no namespace or QObject to associate the QEnum with.